packages/xchain-thorchain/src/client.ts

```diff
@@ -84,8 +82,26 @@ class Client implements ThorchainClient, XChainClient {
     return this.network
   }
 
+  setClientUrl = (clientUrl: string): void => {
+    this.clientUrl = clientUrl
+    this.thorClient = this.getNewThorClient()
+  }
+
   getClientUrl = (): string => {
```

Comment:
In addition to previous comment ^
`getClientUrl = (): {testnetClientUrl: string, mainnetClientUrl: string}`

Reply:
What do you think of following approach? @veado  It returns the client url of current network...
```
  getClientUrl = (): string => {
    return this.getClientUrlByNetwork(this.network)
  }
```